azure/Kqlmagic/kql_magic.py

```diff
@@ -194,6 +197,19 @@ def validate_cloud(self, cloud):
             )
 
 
+    @validate("login_code_destination")
+    def _valid_value_login_code_destination(self, proposal):
+        dest = proposal["value"]
+        if dest != "browser":
+            if not self.check_mailto(dest):
+                message = "The 'login_code_destination' trait of a {0} instance {1}".format(Constants.MAGIC_CLASS_NAME, str(e))
+                raise TraitError(message)
+        return proposal["value"]
+
+    def check_mailto(self, dest):
+        if dest.find("@")!= -1:
```

Comment:
this is too minimal check, please google for a regular expression that checks  email pattern and use it.